src/plugins/cpq-configuration/schema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/amtrack/sfdx-browserforce-plugin/src/plugins/home-page-layouts/schema.json",
```

Comment:
should be `"$id": "https://github.com/amtrack/sfdx-browserforce-plugin/src/plugins/cpq-configuration/schema.json",`